salt/backups/templates/restore_course_assets.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+{% set backupdir = '/mnt/efs/{}'.format(settings.get('directory', 'course_assets')) %}
+
+sudo mkdir -p /mnt/efs
```

Comment:
Yes, because the /mnt directory is owned by root. It's possible that we could mount it somewhere else, but given that this instance is destroyed as soon as the restore is finished I don't think that it would gain us too much.